src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -129,20 +129,9 @@ export default defineComponent({
       ctx.root.$router.replace(route);
     }
 
-    const userCanModifyDandiset: Ref<boolean> = ref(false);
-
-    watchEffect(async () => {
-      // published versions are never editable
-      if (currentDandiset.value?.metadata?.version !== 'draft' || !user.value) {
-        userCanModifyDandiset.value = false;
-      } else if (user.value?.admin) {
-        userCanModifyDandiset.value = true;
-      } else {
-        const { data: owners } = await publishRest.owners(identifier);
-        const userExists = owners.find((owner) => owner.username === user.value?.username);
-        userCanModifyDandiset.value = !!userExists;
-      }
-    });
+    const userCanModifyDandiset: ComputedRef<boolean> = computed(
+      () => store.getters['dandiset/userCanModifyDandiset'],
```

Comment:
My vuex is rusty, but I thought the whole point of direct-vuex was typing this kind of access, i.e.
```
const userCanModifyDandiset = store.getters.dandiset.userCanModifyDandiset;
```
https://github.com/paroi-tech/direct-vuex#use-typed-wrappers-from-outside-the-store mentions using `const store = ctx.root.$store.direct;`, perhaps that is required?

Reply:
We're not using direct-vuex currently. That does look really nice, though. I'll add it and rewrite all the `store` computed vars once I address all of the other needed code changes